models/Diagnostics.json

```diff
@@ -0,0 +1,29 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:azsamples:iotdd:Diagnostics;1",
+    "@type": "Interface",
+    "displayName": "Diagnostics",
+    "description": "Provides functionality to report memory usage and reboot the device",
+    "contents": [
+      {
+        "@type": [
+          "Telemetry"
+        ],
```

Comment:
Since there's only one @type, it would be nice to use the non-array syntax: "@type": "Telemetry"